tox.ini

```diff
@@ -51,7 +51,7 @@ deps =
     isort
 commands =
     codespell {[vars]lib_path}
-    codespell . --skip .git --skip .tox --skip build --skip lib --skip venv --skip .mypy_cache
+    codespell . --skip .git --skip .tox --skip build --skip lib --skip venv* --skip .mypy_cache
```

Comment:
Just curious which cases the `venv*` covers.

Reply:
I have `venv37` and `venv310` and run interactive debugger with different versions